tensorboard/uploader/uploader_test.py

```diff
@@ -1411,6 +1424,57 @@ def test_break_at_tensor_point_boundary(self):
             )
         self.assertEqual(total_points_in_result, point_count)
 
+    def test_strip_large_tensors(self):
+        # Generate test data with varying tensor point sizes. Use raw bytes.
+        event_1 = event_pb2.Event(step=1)
+        event_1.summary.value.add(
+            tag="one", tensor=tensor_pb2.TensorProto(tensor_content=b"\x01\x02")
+        )
+        event_1.summary.value.add(
+            tag="two",
+            tensor=tensor_pb2.TensorProto(
+                tensor_content=b"\x01\x02\x03\x04\x05\x06"
+            ),
+        )
+        event_2 = event_pb2.Event(step=2)
+        event_2.summary.value.add(
+            tag="one", tensor=tensor_pb2.TensorProto(tensor_content=b"\x01\x02")
+        )
+        event_2.summary.value.add(
+            tag="two",
+            tensor=tensor_pb2.TensorProto(
+                tensor_content=b"\x01\x02\x03\x04\x05\x06\x07"
```

Comment:
Similarly "We'll filter this out in both halves of the test below"

Reply:
Done.